assets/blocks/course-outline/module-block/radio-group-control.js

```diff
@@ -0,0 +1,41 @@
+import {
+	__experimentalRadio as Radio,
+	__experimentalRadioGroup as RadioGroup,
+	BaseControl,
```

Comment:
Unfortunately these don't exist in Wordpress 5.3, resulting in an error when opening the block settings. 

An alternative may be just two buttons with toggling an active class, like `is-active: 'in-progress' === previewStatus`. — Which too is different in 5.3, named `is-pressed` vs `is-active` (but here it's safe to just set both)

Reply:
Switched to using ButtonGroup instead. Also took the chance to rename it to something more meaningfull and to switch to booleans.